src/io/fastq.rs

```diff
@@ -3,21 +3,25 @@
 // This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! FastQ reading and writing.
+//! FastQ reading.
```

Comment:
The is a `Writer` class also though?

Reply:
Jep, feel free to contribute an example! ;)

Reply:
This is merely a first step and I quickly made some changes that I thought to be useful. But more documentation is definitely needed and we're planning a Docathon for this. So maybe we can do that during the Docathon.